src/extensions/CommonJSModuleLoader.js

```diff
@@ -10,16 +10,15 @@ export default class CommonJSModuleLoader {
     }
 
     canLoadModule (extensionApi) {
-        return !!extensionApi.serviceDefinition.commonJS;
+        return !!extensionApi.serviceDefinition.module;
```

Comment:
I've moved to using specific keys in service definitions to support multiple different loaders in one container (the key `module` is too generic). I was considering adding a `DefaultModuleLoaderDecorator` which would enable the use of `module` key for that loader, but I figured it's really just a key we've become attached to from habit. Just find and replace all `module` and replace with `commonJS`. I don't think it should be changed here.